install.sh

```diff
@@ -0,0 +1,481 @@
+#!/usr/bin/env bash
+
+# This is a script to quickly install chaos-mesh.
+# This script will check if docker and kubernetes are installed. If local mode is set and kubernetes is not installed,
+# it will use kind or minikube to install the kubernetes cluster according to the configuration.
+# Finally, when all dependencies are installed, chaos-mesh will be installed using helm.
+
+set -e
+
+usage() {
+    cat << EOF
+This script is used to install chaos-mesh.
+Before running this script, please ensure that:
+* have installed docker if you run chaos-mesh in local.
+* have installed Kubernetes if you run chaos-mesh in normal Kubernetes cluster
+USAGE:
+    install.sh [FLAGS] [OPTIONS]
+FLAGS:
+    -h, --help              Prints help information
+OPTIONS:
+    -v, --version           Version of chaos-mesh, default value: latest
+    -l, --local [kind]      Choose a way to run a local kubernetes cluster, supported value: kind,
```

Comment:
Did we need to support in k8s(not local)

Reply:
Yes, In this case, we just to check Kubernetes and install Chaos Mesh.